package.json

```diff
@@ -304,7 +304,7 @@
     "electron-builder": "25.0.0",
     "eslint": "^8.57.0",
     "eslint-config-airbnb-typescript": "^18.0.0",
-    "eslint-config-expensify": "2.0.78",
+    "eslint-config-expensify": "2.0.82",
```

Comment:
I'm not sure why it was so many versions behind... but maybe just an oversight. I didn't see any issues after bumping it.